Assets/Scripts/Enemy/TargetFinder.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using Interfaces;
+using UnityEngine;
+using Utils;
+
+namespace Enemy
+{
+    /// <summary>
+    /// Finds all attackable objects that enter the trigger-collider and gives them to the enemy controller
+    /// </summary>
+    [RequireComponent(typeof(Collider))]
+    public class TargetFinder : MonoBehaviour
+    {
+        [Tooltip("Every layer that will be considered as possible target")] [SerializeField]
+        private LayerMask attackableLayers;
+
+        /// <summary>
+        /// List of all targets that entered the trigger-radius and have a layer of the <see cref="attackableLayers"/> layer mask
+        /// </summary>
+        public readonly List<GameObject> Targets = new List<GameObject>();  //TODO: Should probably be replaced by the abstract class "Damageable"
```

Comment:
Concerning the TODO: Like I said, AttackLogic needs the target's position, so passing Damageable won't be enough.

Reply:
The abstract class should extend from MonoBehaviour or be an component for itself maybe?

Reply:
How about adding something like GetPosition() to Damageable?